core/src/main/java/org/infinispan/health/HealthFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.infinispan.health;
+
+import org.infinispan.health.impl.HealthImpl;
+import org.infinispan.health.impl.jmx.HealthJMXExposerImpl;
+import org.infinispan.health.jmx.HealthJMXExposer;
+import org.infinispan.manager.EmbeddedCacheManager;
+
+/**
+ * An entry point for creating Health API.
+ */
+public class HealthFactory {
+
+    private HealthFactory() {
+    }
+
+    /**
+     * Creates Health API.
+     * <p>
+     *     This method also registers the API in {@link org.infinispan.factories.GlobalComponentRegistry}.
+     * </p>
+     *
+     * @param embeddedCacheManager Cache manager to be monitored.
+     * @return Health API.
```

Comment:
I think every instance of "API" in this javadoc should be replaced with "instance".